blaseball_mike/database.py

```diff
@@ -4,16 +4,29 @@
 """
 import requests
 import requests_cache
+from json.decoder import JSONDecodeError
 
 
 BASE_URL = 'https://www.blaseball.com/database'
 BASE_GITHUB = 'https://raw.githubusercontent.com/xSke/blaseball-site-files/main/data'
 
 
+def _check_response(response):
```

Comment:
we can probably kick this out into the `session` file or something for all common http related helpers